tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,4 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result == False
```

Comment:
Nice job explicitly checking that result is equal to `False` (instead of just checking that it's a falsey value).

How could you make this test more robust? Besides checking that the return value is correct, we should also explicitly check that the `vendor` `inventory` has not been changed. Therefore, we should check that the inventory still has "a", "b", "c" and that `item` was not somehow inserted in there. 

```py
    assert len(vendor.inventory) == 3
    assert "a" in vendor.inventory 
    assert "b" in vendor.inventory 
    assert "c" in vendor.inventory
    assert item not in vendor.inventory